shell/store/plugins.js

```diff
@@ -55,6 +55,7 @@ const driverMap = {
   oci:                             'oracle',
   opentelekomcloudcontainerengine: 'otc',
   oraclecontainerengine:           'oracle',
+  ociocne:                         'oracle',
```

Comment:
Changes under shell need to be demarcated with the following marker comments:
// Added by Verrazzano Start
**** ... code changes ... ****
// Added by Verrazzano End